Makefile

```diff
@@ -59,18 +59,21 @@ ci:
 	$(MAKE)
 	$(MAKE) install
 	$(MAKE) -C test all
+	$(MAKE) -C pkg tag
```

Comment:
nit: ordering wise would it make more sense to check that things `build` *before* we run tests?